### DISCUSSION TITLE
Multi-GPU training

### DISCUSSION THREAD

**Discussion by nicemanis** [idiap/coqui-ai-TTS#52] [Q&A] [answered]
Hi! Is multi-gpu training working/fixed in this fork (e.g. for VITS model)? I remember that there were some issues before (e.g. https://github.com/coqui-ai/TTS/discussions/2584).

**Comment by eginhard** [comment, marked as answer]
No, there are no changes in that part of the code and we haven't needed it so I would assume it's still broken. I'd be happy to merge any fixes, but probably won't have time to look into it myself.
